.azure/gpu-tests-lite.yml

```diff
@@ -62,18 +62,19 @@ jobs:
       displayName: 'set visible devices'
 
     - bash: |
-        set -e
-        PYTORCH_VERSION=$(python -c "import torch; print(torch.__version__.split('+')[0])")
-        python ./requirements/pytorch/adjust-versions.py requirements/lite/base.txt ${PYTORCH_VERSION}
```

Comment:
We need to keep `adjust-versions.py` to adjust the PyTorch version, no?

Reply:
I have the same question

Reply:
ok, I misunderstood you question :)